views/show.blade.php

```diff
@@ -84,12 +84,12 @@
 @stop
 
 @section('css')
-{!! Asset::styles('logviewer') !!}
+    {!! HTML::style(asset('styles/logviewer.css') !!}
```

Comment:
you can't really use the asset function either because it depends on laravel/framework and we don't require that package